ls1intum/Artemis
# Enable C as a programming language for exercises

### Checklist

### Motivation and Context


We want to add support for C as a programming language and use Pytest for testing the submissions.

### Description

As a first step, I copied the Python template and added C as a value to the ProgrammingLanguage enum and dropdown.

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create a new programming exercise
4. Select "C" as the language
5. Submit the solution repository files
6. All tests should be successful
